src/OAM.md

```diff
@@ -92,9 +92,7 @@ lower-priority sprite's "BG and Window over OBJ" attribute.
 
 ## Writing Data to OAM Memory
 
-The recommended method is to write the data to normal RAM first, and to
-copy that RAM to OAM by using the DMA transfer function, initiated
-through DMA register (FF46). Besides, it is also possible to
-write data directly to the OAM area by using normal LD instructions, but this
+The recommended method is to write the data to normal RAM first, then copy that RAM to OAM by using the DMA transfer function, initiated by writing the high byte of the RAM page on which the data is stored to the DMA register ($FF46). While it is also possible to
```

Comment:
We don't usually talk about "RAM page", so this term must not be used.
Suggested: "... by writing the high byte of the address, which must be a multiple of $100, to..."